sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn

```diff
@@ -443,7 +443,7 @@
       <bpmn:extensionElements>
         <zeebe:ioMapping>
           <zeebe:output source="=min(&#10;	max(&#10;	    if (calculatedDataTotalFamilyIncome &#60;= dmnPartTimeAwardFamilySizeVariables.limitAwardCSGDIncomeCap)&#10;		then &#10;			dmnPartTimeAwardAllowableLimits.limitAwardCSGD3OrMoreChildAmount&#10;		else&#10;			max(dmnPartTimeAwardAllowableLimits.limitAwardCSGD3OrMoreChildAmount - ((calculatedDataTotalFamilyIncome - dmnPartTimeAwardFamilySizeVariables.limitAwardCSGDIncomeCap) * dmnPartTimeAwardFamilySizeVariables.limitAwardCSGD3OrMoreChildSlope), 0) * offeringWeeks&#10;        ,&#10;        dmnPartTimeAwardAllowableLimits.limitAwardCSGD3OrMoreChildAmount&#10;    ),&#10;	dmnPartTimeAwardAllowableLimits.limitAwardCSGDAmount&#10;)" target="federalAwardCSGDAmount" />
-          <zeebe:output source="=if (awardEligibilityCSGD = true)&#10;then&#10;  if (calculatedDataTotalRemainingNeed2 &#62; federalAwardCSGDAmount)&#10;  then federalAwardCSGDAmount * offeringWeeks&#10;  else calculatedDataTotalRemainingNeed2 * offeringWeeks&#10;else&#10;0" target="federalAwardNetCSGDAmount" />
```

Comment:
I would recommend to move the common logic outside the exclusive gateway to avoid repetition.

![image](https://github.com/bcgov/SIMS/assets/54600590/509c211f-00d6-4d6f-99c4-b52eaebc615d)